src/components/PopupSignup/PopupSignup.js

```diff
@@ -63,47 +82,55 @@ const PopupSignup = ({
           <input
             id="signup-password"
             name="password"
-            value={passwordValue}
-            onChange={handlePasswordChange}
+            value={formValues.password || ''}
+            onChange={handleInputChange}
             className="popup-with-form__input"
             type="password"
+            minLength="8"
             autoComplete="off"
             placeholder="Enter password"
             required
+            disabled={isLoading}
           />
-          <span className="popup-with-form__error">
-            Input error message
-          </span>
+          {formErrors.password && (
+            <p className="popup-with-form__error">
+              {formErrors.password}
+            </p>
+          )}
         </label>
         <label htmlFor="signup-username" className="popup-with-form__input-group">
           <span className="popup-with-form__input-label">
             Username
           </span>
           <input
             id="signup-username"
-            name="username"
-            value={usernameValue}
-            onChange={handleUsernameChange}
+            value={formValues.username || ''}
```

Comment:
* when I try to sign up I can't write anything into `user name` input. See here http://joxi.ru/VrwxXvlTo08BGm . 


it's because you've forgotten to add attribute `name` here


```js

            name="username"


```


without that your hook `popupForm` doesn't save the info into `formValues`